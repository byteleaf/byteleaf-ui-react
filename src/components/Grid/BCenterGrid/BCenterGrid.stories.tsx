import React from 'react';
import { Button } from '@material-ui/core';

import { BCenterGrid } from './BCenterGrid';

export default {
  title: 'Grid|BCenterGrid',
};

export const BCenterGridStory = () => (
  <BCenterGrid>
    <Button>Submit</Button>
  </BCenterGrid>
);

BCenterGridStory.story = {
  name: 'default',
  parameters: {
    notes: 'A centered grid item',
  },
};
