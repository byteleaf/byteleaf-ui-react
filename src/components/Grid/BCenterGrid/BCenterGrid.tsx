import { Grid, styled } from '@material-ui/core';
import React from 'react';

export const BCenterGrid = styled(Grid)({
  textAlign: 'center',
});
