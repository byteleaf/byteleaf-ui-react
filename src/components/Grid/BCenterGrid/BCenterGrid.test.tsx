import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import { BCenterGrid } from './BCenterGrid';

describe('BCenterGrid', () => {
  it('renders correctly', async () => {
    const { container } = render(
      <BCenterGrid>
        <button>test</button>
      </BCenterGrid>,
    );

    expect(container).toMatchSnapshot();
  });
});
