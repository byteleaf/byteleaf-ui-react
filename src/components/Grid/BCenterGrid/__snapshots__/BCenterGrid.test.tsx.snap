// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BCenterGrid renders correctly 1`] = `
<div>
  <div
    class="MuiGrid-root WithStyles(ForwardRef(Grid))-root-1"
  >
    <button>
      test
    </button>
  </div>
</div>
`;
